src/openforms/appointments/base.py

```diff
@@ -220,3 +224,16 @@ def get_appointment_details_html(self, identifier: str) -> str:
         return render_to_string(
             "appointments/appointment_details.html", {"appointment": details}
         )
+
+    def get_appointment_links_html(self, submission: Submission) -> str:
+
+        token = submission_appointment_token_generator.make_token(submission)
+        cancel_url = reverse(
+            "api:appointments-verify-cancel-appointment-link",
+            kwargs={"token": token, "submission_id": submission.id},
+        )
+
+        return render_to_string(
+            "appointments/appointment_links.html",
+            {"cancel_url": cancel_url},
+        )
```

Comment:
Let's decouple getting the links from the rendering in the template.

This method should just return a dict with the links & any templates should call this method instead and use the links.

I want the option to easily include this information in other places, like and API endpoint used by the SDK without being constrained by the template engine.